lametro/people.py

```diff
@@ -88,8 +100,11 @@ def scrape(self):
                                  classification='committee',
                                  parent_id={'name' : 'Board of Directors'})
 
+                committee_info = web_info.get(org_name, {})
```

Comment:
This may be a weird request, but can we use one of these variable names, `committee_name` or  `org_name`, when we make the `web_info` dict and when we access it here? I got a little cognitive dissonance from the difference. I like `committee_name` myself, because we're just doing committees.